lib/src/ui/pages/home/home_page.dart

```diff
@@ -1,248 +1,38 @@
-          style: Theme.of(context)
-              .textTheme
-              .headline2
-              .copyWith(fontFamily: _Constants.appBarTitleFontFamily),
-        ),
-      ),
-      backgroundColor: Theme.of(context).backgroundColor,
-      actions: [
-        Padding(
-          padding: const EdgeInsets.symmetric(
-            vertical: _Constants.appBarActionsVerticalPadding,
-          ),
-          child: PrimaryIconButton(
-            iconData: Icons.search,
-            onPressed: () {},
-          ),
-        ),
-        Padding(
-          padding: const EdgeInsets.only(
-            top: _Constants.appBarActionsVerticalPadding,
-            bottom: _Constants.appBarActionsVerticalPadding,
-            right: Dimens.pagePadding,
-            left: _Constants.appBarActionsIconsPadding,
-          ),
-          child: PrimaryIconButton(
-            iconData: Icons.settings,
-            onPressed: () {
-              Navigator.push(
-                context,
-                MaterialPageRoute(
-                  builder: (context) => ChangeNotifierProvider(
-                    create: (_) => inject<SettingsNotifier>(),
-                    child: SettingsPage(),
-                  ),
-                ),
-              );
-            },
-          ),
-        ),
-      ],
-    );
-  }
-
-  Widget _buildNewsSectionHeader({
-    @required String title,
-    @required WidgetBuilder routeBuilder,
-  }) {
-    return Padding(
-      padding: const EdgeInsets.symmetric(horizontal: Dimens.pagePadding),
-      child: Row(
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-        children: [
-          Flexible(
-            child: Hero(
-              tag: title,
-              child: Text(
-                title,
-                style: Theme.of(context).textTheme.headline3,
-              ),
-            ),
-          ),
-          PrimaryTextButton(
-            text: Strings.of(context).showAll,
-            onPressed: () => Navigator.push(
-              context,
-              MaterialPageRoute(
-                builder: routeBuilder,
-              ),
-            ),
-          ),
-        ],
-      ),
-    );
-  }
-
-  Widget _buildSecondaryNewsItems(List<Article> articles) => Column(
-        children: articles
-            .toSecondaryNewsListEntities()
-            .take(3)
-            .toList()
-            .map((news) => SecondaryNewsListItem(news: news))
-            .toList(),
-      );
-
-  Widget _errorContent(BuildContext providerContext, HomeNotifierErrorState state) => Center(
-    //TODO use state to display error returned from the notifier
-        child: Container(
-          height: _Constants.retryButtonHeight,
-          width: _Constants.retryButtonWidth,
-          child: PrimaryTextButton(
-            onPressed: () => providerContext.read<HomeNotifier>().fetchHomeArticles(),
-            text: Strings.of(context).retry,
-          ),
+  Widget build(BuildContext context) => Scaffold(
```

Comment:
zastanawiam się czy to jest ok, że tutaj mamy Scaffold, a w body mamy taby (FeedPage i ProfilePage), które mają jeszcze swój Scaffold 🤔

Reply:
został tylko na HomePage

Reply:
super, jedyne co to jakoś appbar trzeba inaczej ograć, bo ten jest dość mocno pod dashboard, jeszcze nie określaliśmy jak szczegółowo będzie wyglądał profil, ale w ciemno można stwierdzić, że takiego searchButtona na pewno nie będzie (ogólnie wydaje mi się, że nic prócz ustawień z poziomu profilu nie będzie na appbarze w profilu)

Reply:
no zerknij

Reply:
no póki co nie przychodzi mi do głowy jakieś ładniejsze rozwiązanie, jak dojdzie jakaś logika to pewnie się wydzieli do osobnego pliku i trzeba będzie wyciągnąć changeNotifiery wyżej w drzewie ;) nawet widzę, że nawet ktoś założył issue, żeby był mniejszy coupling :D 
https://github.com/flutter/flutter/issues/36461